client/src/api/unculledZonesSlice.ts

```diff
@@ -0,0 +1,36 @@
+import { RootState } from "../app/store";
+import { gameLoaded, gameUnloaded } from "./actions";
+import { UnculledZone } from "./liberationApi";
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+
+interface UnculledZonesState {
+  zones: UnculledZone[];
+}
+
+const initialState: UnculledZonesState = {
+  zones: [],
+};
+
+export const unculledZonesSlice = createSlice({
+  name: "unculledZonesState",
+  initialState,
+  reducers: {
+    updated: (state, action: PayloadAction<UnculledZone[]>) => {
+      state.zones = action.payload;
+    },
```

Comment:
This should trigger the front-end to redraw, however in order for this to work you must reference it using `selectUnculledZones` using the "AppSelector" as is done on line 32 of `CullingExclusionZones.tsx`